### ISSUE TITLE
electrum-grs: fails to build

### ISSUE THREAD

**Issue by lourkeur** [NixOS/nixpkgs#207746]
### Steps To Reproduce
1. build `electrum-grs` with unfree packages enabled.

### Build log
```
ERROR: Could not find a version that satisfies the requirement protobuf<4,>=3.12 (from electrum-grs) (from versions: none)
ERROR: No matching distribution found for protobuf<4,>=3.12
```

### Additional context
Other Electra use this patchPhase. It's dodgy but it has worked so far.
```sh
    # make compatible with protobuf4 by easing dependencies ...
    substituteInPlace ./contrib/requirements/requirements.txt \
      --replace "protobuf>=3.12,<4" "protobuf>=3.12"
```

### Notify maintainers
@gruve-p

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.83, NixOS, 22.11 (Raccoon), 22.11.20221224.dac57a4`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.11.1`
 - nixpkgs: `/etc/nix/channels/nixos`
```

**Comment by gruve-p** [comment]
Fix in #207752

**Comment by gruve-p** [comment]
Fix in: #207834
